src/components/SkipLink/SkipLink.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import SkipLink from './SkipLink';
+
+const meta = {
+  title: 'Components/SkipLink',
+  component: SkipLink,
+  argTypes: {
+    destination: {
+      controlType: 'text'
+    },
+    destinationId: {
+      controlType: 'text'
+    }
+  }
```

Comment:
Since we moved descriptions to code, can we remove the manually defined control types?